docs/thirdparty-openapi3-snippets.yaml

```diff
@@ -69,6 +69,7 @@ paths:
                         - OTP
                         - QRCODE
                         - U2F
+                        - PKC
```

Comment:
do we really need PKC?

Reply:
nope. err'd too much on the side of caution and defaulted to addition.